cogni-libs/src/test/java/zone/cogni/libs/core/utils/DateFormatsTest.java

```diff
@@ -67,12 +67,18 @@ public void fixed_date_as_xsd_datetime_format() {
     String oldBeforePlus = StringUtils.substringBefore(oldFormatter, "+");
     String newBeforePlus = StringUtils.substringBefore(newFormatter, "+");
 
+    String oldBeforeZ = StringUtils.substringBefore(oldBeforePlus, "Z");
+    String newBeforeZ = StringUtils.substringBefore(newBeforePlus, "Z");
+
     String oldAfterPlus = StringUtils.substringAfterLast(oldFormatter, "+");
     String newAfterPlus = StringUtils.substringAfterLast(newFormatter, "+");
 
+    String oldAfterZ = StringUtils.substringAfterLast(oldAfterPlus, "Z");
+    String newAfterZ = StringUtils.substringAfterLast(newAfterPlus, "Z");
```

Comment:
mix between zulu and non-zulu tests

split up and make a test with zulu and non-zulu timezones 
you can use this (first get default timezone and set it back after the test)
`   TimeZone.setDefault(TimeZone.getTimeZone("Europe/Paris"));
    TimeZone.setDefault(TimeZone.getTimeZone("UTC"));`
    
Then you can also test more precisely on endswith "+02:00" or "Z"